src/library_html5.js

```diff
@@ -2,6 +2,8 @@
  * @license
  * Copyright 2014 The Emscripten Authors
  * SPDX-License-Identifier: MIT
+ * SPDX-FileCopyrightText: Portions Copyright 2023 Siemens
+ * Modified on February 2023 by Siemens and/or its affiliates to improve memory64 support
```

Comment:
We tend not to put individual names in our files like this.   Can you add yourself to `AUTHORS` instead perhaps?